packages/runner/src/types/tasks.ts

```diff
@@ -210,15 +210,22 @@ export type TestAPI<ExtraContext = {}> = ChainableTestAPI<ExtraContext> & Extend
       K extends keyof ExtraContext ? ExtraContext[K] : never }>
 }
 
-export type Use<T> = (value: T) => Promise<void>
+export interface FixtureOptions {
+  /**
+   * Whether to automatically set up current fixture, even though it's not being used in tests.
+   */
+  auto?: boolean
+}
+
+export type Use<T> = (value?: T) => Promise<void>
```

Comment:
Is it okay to change it to optional `value?: T`? I thought keeping it non-optional is correct since otherwise it might allow something like:

```ts
const myTest = test.extend<{ someNumber: number }>({
  someNumber: async ({}, use) => {
    await use() // no type error but it should be `use(1234)`?
  }
})
```

Reply:
I think you are right, and I just checked that the `value` is also non-optional in Playwright, so I'll revert it to be required, thanks!